overlays/openstack.yaml

```diff
@@ -0,0 +1,28 @@
+applications:
+  easyrsa:
+    charm: easyrsa
+    channel: stable
+    num_units: 1
+    constraints: cpu-cores=1 mem=4G root-disk=16G
+    base: ubuntu@22.04
```

Comment:
🤮 

No, let's not.  I don't want to have easyrsa in our bundles.  This should be a certificate authority we try to not use anymore.